archinstall/lib/mirrors.py

```diff
@@ -450,8 +450,8 @@ def get_status_by_region(self, region: str, speed_sort: bool) -> list[MirrorStat
 		# Local mirrors lack this data and can be modified manually before-hand
 		# Or reflector potentially ran already
 		if self._fetched_remote and speed_sort:
-			# original return
-			return sorted(region_list, key=lambda mirror: (mirror.score, mirror.speed))
+			# Sort by speed descending (higher is better) - fastest mirrors first
+			return sorted(region_list, key=lambda mirror: -mirror.speed)
```

Comment:
Could we move this to a separate PR? Just to not fix two things in one, in case one needs to be reverted :)